tests/testgroup70.py

```diff
@@ -1003,3 +1003,37 @@ def runTest(self):
         flow_match_test(self, config["port_map"], pkt=pkt, exp_pkt=exp_pkt, 
                         action_list=acts, max_test=2)
 
+class Grp70No240(base_tests.SimpleProtocol):
+    """
+    Ordering not possible - cannot process in order specified
+    If a switch cannot process the action list for any  flow mod message in the order specied,
+    it must return an Error with error.type OFPET_FLOW_MOD_FAILED and error.code OFPFMFC_UNSUPPORTED or ofp.OFPFMFC_EPERM and reject the flow """
+
+    def runTest(self):
+        logging = get_logger()
+        logging.info("Running FlowModFailed Ordering not possible Grp70No240 test")
+
+        rc = delete_all_flows(self.controller)
+        self.assertTrue(rc != -1, "Error installing flow mod")
+        self.assertEqual(do_barrier(self.controller), 0, "Barrier failed")
+
+        msg = message.flow_mod()
+        act1 = action.action_output()
+        act2 = action.action_set_tp_src()
+        act1.port = 99
```

Comment:
I think `act1.port` should be a valid configured openflow dataplane `port` because this scenario may also trigger some other `error code` 
@jonstout What do you think?